pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -754,38 +762,6 @@ func updateLispConfiglets(ctx *zedrouterContext, legacyDataPlane bool) {
 // Wrappers around handleCreate, handleModify, and handleDelete
 
 // Determine whether it is an create or modify
-func handleAppNetworkConfigModify(ctxArg interface{}, key string, configArg interface{}) {
-
-	ctx := ctxArg.(*zedrouterContext)
-	config := cast.CastAppNetworkConfig(configArg)
-	log.Infof("handleAppNetworkConfigModify(%s-%s)\n", config.DisplayName, key)
-
-	// If this is the first time, update the timer for GC
-	if ctx.receivedConfigTime.IsZero() {
-		log.Infof("triggerNumGC")
-		ctx.receivedConfigTime = time.Now()
-		ctx.triggerNumGC = true
-	}
-
-	status := lookupAppNetworkStatus(ctx, key)
-	if status == nil {
-		handleAppNetworkCreate(ctx, key, config)
-		status = lookupAppNetworkStatus(ctx, key)
-	} else {
-		doAppNetworkConfigModify(ctx, key, config, status)
-	}
-	// on error, relinquish the acquired resource
```

Comment:
You appear to have lost line 777-787. I suspect that needs to be applied in both the create and modify handlers.